tests/strategies/test_exit.py

```diff
@@ -185,3 +185,39 @@ def test_exit_by_trailing_stop(market, signal, entries):
     )
 
     pd.testing.assert_frame_equal(trades, expected)
+
+
+def test_exit_at_loss_and_gain(market, signal, entries):
+    max_holding_time = pd.Timedelta("3min")
+    gain_threshold = 100
```

Comment:
Please add another test when gain_threshold is small.

Reply:
Sure, I'll add it.